.git-blame-ignore-revs

```diff
@@ -0,0 +1,8 @@
+# Format the world using Prettier and ESLint
+f042747aa077cda1a01a2b72ae8178c3ac04416a
```

Comment:
Are these commit change ids? Why are there 6? I thought we just had one format the world change? Did we not squash it?

Reply:
oh yea I don't know we need to squash it before.